src/microsoft/office/license/user/add/microsoft-office-license-user-add.controller.js

```diff
@@ -42,6 +45,15 @@ angular.module("Module.microsoft.controllers").controller("MicrosoftOfficeLicens
         this.getLicenses();
     }
 
+    getLicensePrice () {
+        this.loaders.licensePrice = true;
+
+        this.licenseService.getLicensePrice(this.user.licence)
+            .then((licensePrice) => { this.licensePrice = licensePrice; })
+            .catch(() => { this.licensePrice.text = this.$scope.tr("microsoft_office_license_add_user_license_price_error"); })
```

Comment:
The text for the license price is an error is there has been an error ? I think it would be better to have a proper variable containing the error text

Reply:
An error while retrieving the license price does not stop the user from going ahead with adding the user. Hence the error is displayed in the place where the price should be displayed. Please let me know if this has to be handled differently.

Reply:
Feedback accepted :). A new variable now holds the error text.

Reply:
Alright, good job !